templates/graphdb/configmap-properties.yaml

```diff
@@ -20,6 +20,42 @@ data:
     {{- if eq (int .Values.replicas) 1}}
     graphdb.external-url={{ include "graphdb.external-url" . }}
     {{- end }}
+    {{- with .Values.cluster.config.tls }}
+    {{- if ne .mode "NONE" }}
+    graphdb.raft.security.mode={{ .mode  }}
+    {{- if .privateKey.secretName }}
+    graphdb.raft.security.certificateKeyFile=/opt/graphdb/home/conf/private-key.pem
```

Comment:
Anything in the conf folder ends up in the server reports so this would leak the private keys.
Choose another directory, maybe `/etc/graphdb/`, @yaskoo WDYT?